src/styles/common/_mixins.scss

```diff
@@ -203,7 +202,6 @@
 		background-color: $tableBodyBackground;
 		border-right: 0;
 		border-left: 0;
```

Comment:
i think this breaks aligning in first column of verification table

Reply:
Fixed